pkg/operations/storage/notification.go

```diff
@@ -153,7 +153,7 @@ type NotificationOps struct {
 	// Topic is the environment variable containing the PubSub Topic being
 	// subscribed to's name. In the form that is unique within the project.
 	// E.g. 'laconia', not 'projects/my-gcp-project/topics/laconia'.
-	Topic string `envconfig:"PUBSUB_TOPIC_ID" required:"true"`
+	Topic string `envconfig:"PUBSUB_TOPIC_ID" required:"false"`
```

Comment:
why false? And if it is not required then don't you want to check during operations.ActionCreate case that it is given?

Reply:
not needed for the delete. The call will fail without it and the error messages are logged, so figured that was fine.
Anyways, added a check for it.